en/guides/students/login-and-settings/index.md

```diff
@@ -1,199 +1,79 @@
 ---
-title: "[nl] Aanmelden & gebruikersinstellingen"
-description: "Tutorial: aanmelden en gebruikersinstellingen"
+title: "Signing Up & User Settings"
+description: "Tutorial: signing up and user settings"
 order: 2
 ---
 
-::: warning Sorry
-For now, this page is only available in Dutch. Sorry!
-:::
-
-# Aanmelden & gebruikersinstellingen
-> Op deze pagina vind je alle informatie zodat je als student een gebruikersaccount op Dodona kan aanmaken en die kan personaliseren.
 
-**Inhoudsopgave**
-[[toc]]
+# Signing Up & User Settings
+On this page, you will find all the information you need to create and personalize a user account on Dodona as a student.
 
-## Selecteren van de taal
+## Selecting the Language
 
-Je kan Dodona gebruiken in het Nederlands of het Engels. Gebruik hiervoor het taalmenu aan de rechterkant van de navigatiebalk (bovenaan elke pagina).
+You can use Dodona in English or Dutch. Use the language menu on the right side of the navigation bar (at the top of every page) for this purpose.
 
 ![image](./choose_language.png)
 
-Als je een taal selecteert terwijl je aangemeld bent dan wordt die ook als voorkeurstaal ingesteld in je [gebruikersprofiel](https://dodona.be/profile). Op die manier wordt bij aanmelden automatisch je voorkeurstaal geselecteerd.
+If you select a language while logged in, it will also be set as your preferred language in your [user profile](https://dodona.be/profile). This way, your preferred language will be automatically selected upon login.
 
-::: tip Wordt alles vertaald?
+::: tip Is Everything Translated?
 
-De vaste componenten van Dodona (menubalken, cursusstructuur, mededelingen, ...) worden altijd weergegeven in de geselecteerde taal.
+The fixed components of Dodona (menu bars, course structure, notifications, etc.) are always displayed in the selected language.
 
-De naam en beschrijving van een [cursus](../courses/) worden weergegeven in de taal zoals de lesgever ze heeft ingegeven. Deze worden niet aangepast aan je taalvoorkeuren.
+The name and description of a [course](../courses/) are displayed in the language as entered by the instructor. These are not adapted to your language preferences.
 
-De naam en beschrijving van een [oefening](../exercises/#navigeren-naar-een-oefening) worden steeds in je voorkeurstaal weergegeven indien beschikbaar. Niet alle oefeningen zijn namelijk in beide talen beschikbaar.
+The name and description of an [exercise](../exercises/#navigating-to-an-exercise) are always displayed in your preferred language if available. Not all exercises are available in both languages.
 
 :::
 
-## Aanmelden
+## Signing Up
```

Comment:
sign up = registreren; sign in = aanmelden

Reply:
I know.  And `sign up` is not required for Dodona.
But this can be confusing to some users. Which is why I wrote the guide as 'if you want to sign up, click sign in'
but maybe that has made it even more confusing.

If you want I can also replace all remaining mentions of `sign up` with `sign in` and then add a notice: there is no need to sign up for Dodona, you cal immediately sign in and start using the platform.

Reply:
I think the guide is clear enough on that? I you want to sign up, simply click the sign in button and Dodona wil automatically create an account for you.